src/common/src/blobs.cpp

```diff
@@ -95,8 +98,11 @@ int Blobs::runlengthAnalysis(Qqueue *qq)
         res = handleSegment(row, qval.m_col_start, qval.m_col_end - 1);
     }
 
-    if ((qval.m_col_start & QVAL_VAL_MASK) == QVAL_FRAME_ERROR) // error code, queue overrun
+    if ((qval.m_col_start & QVAL_VAL_MASK) == QVAL_FRAME_ERROR || // return error if queue overrun
```

Comment:
Can you put another set of parenthesis around the first logic stmt?

Reply:
Done